lnrpc/routerrpc/router_server.go

```diff
@@ -1506,6 +1514,129 @@ func (s *Server) HtlcInterceptor(stream Router_HtlcInterceptorServer) error {
 	).run()
 }
 
+// XAddLocalChanAliases is an experimental API that creates a set of new
+// channel SCID alias mappings. The list of successfully created mappings is
+// returned. This is only a locally stored alias, and will not be communicated
+// to the channel peer via any message. Therefore, routing over such an alias
+// will only work of the peer also calls this same RPC on their end.
+func (s *Server) XAddLocalChanAliases(_ context.Context,
+	in *AddAliasesRequest) (*AddAliasesResponse, error) {
+
+	resp := &AddAliasesResponse{
```

Comment:
nit: looks like we're not using this until the very end. So can just inline-initialize it there, so we don't need the empty pre-allocation.

Reply:
Same in the delete and listaliases (pre-existing) RPC methods.